pvm/ts/src/commands/balanceCommand.ts

```
@@ -1,9 +1,10 @@
-import { AccountManagement, getAccountManagementInstance } from "../state/accountManagement";
+import { getAccountManagementInstance } from "../state/index";
+import { IAccountManagement } from "../state/interfaces";
 import { signResult } from "./abstractSignedCommand";
 import { ISignedCommand, ISignedResponse } from "./interfaces";
 
 export class BalanceCommand implements ISignedCommand<BigInt> {
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)